operator/internal/values/translation/translation.go

```diff
@@ -416,3 +416,12 @@ func GetGlobalNetwork(s *platform.GlobalNetworkSpec) *ValuesBuilder {
 	}
 	return &sv
 }
+
+// GetConfigAsCode converts *platform.GetConfigAsCodeSpec into *ValuesBuilder
+func GetConfigAsCode(c *platform.ConfigAsCodeSpec) *ValuesBuilder {
+	sv := NewValuesBuilder()
+	if c.Enabled != nil {
```

Comment:
Just want to clarify: the behavior here is that when `configAsCode` is not configured in the CR, then the Helm chart defaults will end up being used, right?

Reply:
If `configAsCode` is not set in the CR, then the values applied to the helm chart will effectively be `configAsCode.enabled=true` (coming from Helm chart defaults) and `configAsCode.installCRD=false` (coming from `operator/internal/central/values/translation/base-values.yaml`)